ansible/inventories/devnet-0/group_vars/nethermind.yaml

```diff
@@ -37,8 +38,10 @@ nethermind_container_command_extra_args:
   - --EthStats.Secret={{ ethstats_secret }}
   - --EthStats.Server=wss://{{ ethstats_url }}/api/
   - --log=DEBUG
+  - --Metrics.Enabled=true
+  - --Metrics.NodeName="{{ ethereum_network_name }}-{{ inventory_hostname }}"
```

Comment:
to be checked by @parithosh or @barnabasbusa  (or anyone else in knowledge)

Will it generate a name like "pectra-devnet-2-nethermind-lighthouse" as an example?